packages/inscription/package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "@mdip/inscription",
+  "version": "1.2.0",
+  "description": "MDIP Inscription",
+  "type": "module",
+  "main": "./dist/cjs/index.cjs",
+  "module": "./dist/esm/index.js",
+  "types": "./dist/types/index.d.ts",
+  "files": [
+    "dist"
+  ],
+  "exports": {
+    ".": {
+      "import": "./dist/esm/index.js",
+      "require": "./dist/cjs/index.cjs",
+      "types": "./dist/types/index.d.ts"
+    },
+    "./types": {
+      "types": "./dist/types/types.d.ts"
+    }
+  },
+  "typesVersions": {
+    "*": {
+      "inscription": [
+        "./dist/types/inscription.d.ts"
+      ],
+      "types": [
+        "./dist/types/types.d.ts"
+      ]
+    }
+  },
+  "scripts": {
+    "clean": "rm -rf dist",
+    "build:esm": "tsc -p tsconfig.json",
+    "bundle:cjs": "rollup -c rollup.cjs.config.js",
+    "build": "npm run clean && npm run build:esm && npm run bundle:cjs"
+  },
+  "keywords": [],
+  "author": "Peter Bushnell <bushsolo@gmail.com>",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/KeychainMDIP/kc.git"
+  },
+  "devDependencies": {
+    "@rollup/plugin-commonjs": "^28.0.3",
+    "@rollup/plugin-node-resolve": "^16.0.1",
+    "rollup": "^4.40.2"
+  },
+  "dependencies": {
```

Comment:
@jahatch512 can you confirm these dependencies are available for mobile apps?

Reply:
we may need "crypto"native polyfills